services/movie.py

```diff
@@ -0,0 +1,30 @@
+from db.models import Movie, Genre, Actor
+
+
+def get_movies(genres_ids: list[int], actors_ids: list[int]) -> Movie:
+    queryset = Movie.objects.all()
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+    return queryset.distinct()
+
+
+def get_movie_by_id(movie_id:int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(movie_title: str,
+                 movie_description: str,
+                 genres_ids: int=None,
+                 actors_ids: int=None,
+                 ) -> Movie:
+    movie=Movie.objects.create(title=movie_title, description=movie_description)
+    if genres_ids:
+        genres=Genre.objects.filter(id__in=genres_ids)
+        movie.genres.set(genres)
+
+    if actors_ids:
+        actors=Actor.objects.filter(id__in=actors_ids)
+        movie.genres.set(actors)
```

Comment:
There is a logical error here. The `actors` should be set to the `actors` field, not the `genres` field. Change `movie.genres.set(actors)` to `movie.actors.set(actors)`.